src/layout/Likert/LikertComponent.module.css

```diff
@@ -16,13 +21,22 @@
   width: 100%;
 }
 
+.likertRadioButton {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.likertRadioButton label {
```

Comment:
The `label` has a gap that comes from the design system. This made the radio button ever so slightly not centered in the table cell.